app/scripts/controllers/deployment.js

```diff
@@ -112,6 +124,7 @@ angular.module('openshiftConsole')
             $scope.loaded = true;
             $scope.deployment = deployment;
             updateHPAWarnings();
+            EnvironmentService.toggleReadonlyForContainerEnvsValueFrom(_.get($scope.deployment, 'spec.template.spec.containers'), envSecretAndConfigMapAreReadonly);
```

Comment:
Other than the gorp, looking @ the toggle again per our discussion to see if we can clean up.